react-components/src/data-providers/core-dm-provider/filterNodesByMappedTo3d.ts

```diff
@@ -31,7 +31,8 @@ export async function filterNodesByMappedTo3d(
   nodes: InstancesWithViewDefinition[],
   revisionRefs: DmsUniqueIdentifier[],
   _spacesToSearch: string[],
-  fdmSdk: FdmSDK
+  fdmSdk: FdmSDK,
+  includeIndirectRelations: boolean
```

Comment:
What do you think about adding a default value here?

Reply:
I don't think I want it - would rather have this explicit, with no mystical default values giving unexpected results at some point